erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -656,8 +656,7 @@ def get_item_details(self, args=None, for_update=False):
 			'has_batch_no'			: item.has_batch_no,
 			'sample_quantity'		: item.sample_quantity
 		})
-		for d in [["Account", "expense_account", "default_expense_account"],
-			["Cost Center", "cost_center", "cost_center"]]:
+		for d in [["Cost Center", "cost_center", "cost_center"]]:
```

Comment:
`cost center`  is also not required, as now we have company specific defaults in Item master.